### PR TITLE
Add translation for get example drawings endpoint

### PR DESCRIPTION
**Pull request by sondrefj**
# Changes :gem:
_Add dictionary in models that can be used to translate a label from Norwegian to English_
* This is currently used for the "getExampleDrawings"-endpoint

# Notes :book: :pencil2:
_Messed up with the commits on this one, so don't worry too much about that heh.._

### CODE REVIEW

File: src/webapp/models.py [hunk 1]
```diff
@@ -565,6 +565,17 @@ def get_translation_dict():
     except Exception as e:
         raise Exception("Could not read Labels table: " + str(e))
 
+def get_norwegian_to_english_dict():
+    """
+        Reads all labels from database and create dictionary
+        that translates from norwegian to english.
+    """
+    try:
+        labels = Labels.query.all()
```
Comment (ComputasAlex) [inline src/webapp/models.py:574]: Not sure if it's a good idea quering for all labels: Made another query that works pretty good and only returns the relevant label:

```
def to_english(norwegian_label):
    """
        Reads the labels table and return the english primary key of the norwegian word
    """
    try:
        query = Labels.query.filter(Labels.norwegian == norwegian_label)[0]
        return str(query.english)

    except AttributeError as e:
        raise AttributeError(
            "Could not find translation in Labels table: " + str(e)
        )
```

Comment (ComputasAlex) [reply src/webapp/models.py:574]: Should be fine as long as we have a one-to-one mapping in the db which is the case for now and probably looking forward as well

File: src/webapp/api.py [hunk 2]
```diff
@@ -243,6 +243,11 @@ def get_n_drawings_by_label():
     """
     n = request.args.get("n", default=None, type=int)
     label = request.values["label"]
+    lang = request.values["lang"]
+
+    if lang == "NO":
+        translations = models.get_norwegian_to_english_dict()
+        label = translations[label]
```
Comment (ComputasAlex) [inline src/webapp/api.py:250]: If change in models.py is accepted this simply becomes:

```
if lang == "NO":
  label = models.to_english(label)
```

**Review by ComputasAlex — CHANGES_REQUESTED** [review]
Suggest we change query to not fetch all labels in backend from db. See code suggestion in comments

File: src/webapp/models.py (hunk 1 above)
Comment (sondrefj) [reply src/webapp/models.py]: Made the changes now. And it worked perfectly. Good suggestion!

**Review by ComputasAlex — APPROVED** [review]

**Pull request merged**